testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/jetbrains/StatementCollectorTest.java

```diff
@@ -33,7 +34,17 @@ public void setUp() throws Exception {
     }
 
     public void testSingleQuery() {
-        parametersService.registerParametersProvider(() -> EMPTY_PARAMETERS);
+        parametersService.registerParametersProvider(new ParametersProvider() {
+            @Override
+            public String getParametersJson() {
+                return EMPTY_PARAMETERS;
+            }
+
+            @Override
+            public String getLocalParametersJson() {
```

Comment:
Local parameters is bad name. Can't get from name what "local" means, because all of them ar local.
Just use `per file` os something like this.

Reply:
fixed in [d787900e](https://github.com/ex3m1024/jetbrains-plugin-graph-database-support/commit/d787900e73bc18d48e9b0ac186e839f178cb809b)